x/ibc/handler.go

```diff
@@ -20,8 +19,7 @@ func NewHandler(k Keeper) sdk.Handler {
 			return client.HandleMsgCreateClient(ctx, k.ClientKeeper, msg)
 
 		case client.MsgUpdateClient:
-			return client.HandleMsgUpdateClient(ctx, k.ClientKeeper, msg)
-
+			return &sdk.Result{}, nil // XXX: add events either here or in ante
```

Comment:
why did we move this msg to the ante handler?

Reply:
for atomic multimsg txs